src/site/markdown/develop/monitors.md

```diff
@@ -83,3 +83,25 @@ metrics:
       stateSet: # <string-array>
       output: # <enum> | possible values [ Gauge, Counter, UpDownCounter ] | Optional | Default: UpDownCounter
 ```
+
+### Overriding connector root level metrics
+
+This section shows how to override the connector root level metrics with the monitor job metrics.
+
+```yaml
+connector:
+# ...
+
+monitors:
+  <monitorType>: # <object>
+    <job>: # <object> | <job> key possible values [ discovery, collect, simple]
+      type: # <string> | Only for collect <job> | possible values [ multiInstance, monoInstance ]
+      keys: # <string-array> | Only for collect <job> with multiInstance type | Default: [ id ]
+      metrics:
```

Comment:
Sorry but that’s not accurate, the `metrics` section isn’t meant to be defined under the `job` according to the engine’s design.